src/core_plugins/elasticsearch/lib/create_data_cluster.js

```diff
@@ -0,0 +1,30 @@
+export default function (server) {
+  const config = server.config();
+  const esPlugins = server.plugins.elasticsearch;
+  const Logger = server.plugins.elasticsearch.ElasticsearchClientLogging;
+
+  class DataClientLogging extends Logger {
```

Comment:
And the same comment as before about exporting this separately so it can be unit tested.